src/editors/selectEditor.js

```diff
@@ -66,6 +68,10 @@ class SelectEditor extends BaseEditor {
   close() {
     this._opened = false;
     this.select.style.display = 'none';
+
+    if (hasClass(this.select, 'ht_editor_show')) {
```

Comment:
Maybe all those used classed could be moved to constants on the top of the file or event to the helper?